README.md

```diff
@@ -35,7 +35,8 @@ Once the `conda-forge` channel has been enabled, `mss` can be installed with:
 
     $ conda create -n mssenv mamba
     $ conda activate mssenv
-    $ mamba install mss
+    (mssenv) $ mamba install mss=6.0.0 python
+    (mssenv) $ mss
```

Comment:
instead of the hardcoded 6.0.0 I would like to use the recent tag of the branch, But I've no idea if this can be filled in. If there is a var which we can use.

on command line this can be done by e.g.
https://newbedev.com/how-to-get-the-latest-tag-name-in-current-branch-in-git

If we commit a hardcoded version string this file needs on any further release a change too.

Reply:
Doesn't `mamba install mss python` suffice?
Once the python fixation is gone, it should look for the appropriate version by itself.

Reply:
for the test environment I needed to add always python only for any new single package added
https://github.com/Open-MSS/MSS/pull/1294/files#diff-7dc87d4394e1756c519dbfd0b80d3b31377f643f0bc25d3ed807ce8a3794023d

Reply:
on windows the install script fetches now the latest version, so no need to change this.

Reply:
`mamba install mss python` works, We should not force the version number in the README here.

Reply:
> 

Ah interesting, so it works because mss finds the appropriate version by itself and we don't need to specify it.